docs/devcontainers/devcontainer-cli.md

```diff
@@ -29,24 +29,15 @@ When tools like VS Code and Codespaces detect a `devcontainer.json` file in a us
 
 This CLI can either be used directly or integrated into product experiences, similar to how it's integrated with Dev Containers and Codespaces today. It currently supports both a simple single container option and integrates with [Docker Compose](https://docs.docker.com/compose/) for multi-container scenarios.
 
-The CLI is available for review in a new [devcontainers/cli](https://github.com/devcontainers/cli) repository and you can read more about its development in [this issue in the spec repo](https://github.com/devcontainers/spec/issues/9).
+The CLI is available in the [devcontainers/cli](https://github.com/devcontainers/cli) repository.
 
 ## System requirements
 
-To use the VS Code dev container CLI, you'll need the following on your system or CI/DevOps environment:
+You can quickly try out the CLI in just a few simple steps, either by installing its npm package or building the CLI repo from sources.
```

Comment:
Given VS Code now ships a pre-built version of the CLI that also adds the "open" command, I wonder if we shouldn't push people towards using the "Install Dev Container CLI" command first.  We can then point out that there's other options for using the CLI elsewhere:

1. Install via npm - which then has these requirements right now.
2. Use the GitHub Action or Azure DevOps Task

Reply:
Added info on using the install command from the Dev Containers extension, and restructured a bit to add the Action and ADO task.